ios/InstagramStories.m

```diff
@@ -22,7 +22,7 @@ @implementation InstagramStories
 
 - (void)backgroundImage:(NSData *)backgroundImage attributionURL:(NSString *)attributionURL {
     // Verify app can open custom URL scheme, open if able
-    
+
```

Comment:
Nothing in this file changed except whitespace, which unnecessarily slows down review